.github/workflows/ci.yml

```diff
@@ -33,8 +33,7 @@ jobs:
           - shard.override.yml
         crystal_version:
           - 1.0.0
-          - 1.1.0
-          - 1.2.0
+          - 1.3.0
```

Comment:
```suggestion
          - latest
```

Reply:
I thought the same thing.... I'm wondering though, would this ever be an issue where maybe a Crystal version is released, and this breaks something? Now thinking about it, once 2.0 drops, we will have to change all of this to not be 2.0 until we know for sure Lucky is supported. 🤔  I guess that's a problem for a later day

Reply:
Since we're also running specs on nightly, we would know about it ahead of time